src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -49,30 +48,46 @@ const otherToolsLink = (
   </p>
 );
 
+const headingStyle = {
+  fontWeight: '700',
+  fontFamily: 'Bitter, Georgia, Cambria, Times New Roman, Times, serif',
+  lineHeight: '1.3',
+  clear: 'both',
+};
```

Comment:
I think you could create this style pretty easily using the font utilities - https://design.va.gov/utilities/font-size